config/v1/types_infrastructure.go

```diff
@@ -337,7 +337,7 @@ type AzurePlatformStatus struct {
 }
 
 // AzureCloudEnvironment is the name of the Azure cloud environment
-// +kubebuilder:validation:Enum="";AzurePublicCloud;AzureUSGovernmentCloud;AzureChinaCloud;AzureGermanCloud
+// +kubebuilder:validation:Enum="";AzurePublicCloud;AzureUSGovernmentCloud;AzureChinaCloud;AzureGermanCloud;AzureStackCloud;
```

Comment:
Is it valid for this to end in a `;`?

Reply:
Nice catch. I don't think so, so I fixed it, despite the passing tests, generated docs.